release/xgboost_tests/workloads/ft_small_elastic.py

```diff
@@ -0,0 +1,50 @@
+"""Fault tolerance test (small cluster, elastic training)
+
+In this run, two training actors will die after some time. It is expected that
+in both cases xgboost_ray stops training, but continues right away with the
+remaining three actors. Shortly after, the actors will be restarted and
+re-integrated into the training loop. Training should finish with all four
```

Comment:
This should be remaining two actors right, not three?

Reply:
Right, I'll fix it, thanks!

Reply:
Actually no, the actors die at different points in time, so it will continue with 3 actors, then restart an actor, then continue with 4, then it another one will die, etc.